packages/components/tests/dummy/app/styles/app.scss

```diff
@@ -86,20 +89,49 @@ body.dummy-app {
             min-height: 84.75vh;
             width: 1024px;
         }
+
         .dummy-footer {
-            padding: 0 2rem;
-            background-color: black;
-            color: white;
-            .dummy-paragraph {
-                padding: 0;
-                margin: 0;
+            background-color: #f1f2f3;
+            box-sizing: border-box;
+            display: flex;
+            flex-flow: row wrap;
+            margin: auto;
+            padding: 0;
+            width: 100%;
 
-                .dummy-link {
-                color: white;
-                display: flex;
-                align-content: center;
-                padding: 1rem;
-                width: fit-content;
+            .dummy-row-wrapper {
+                padding: 2rem;
+                margin: 0;
+                width: 100%;
+                max-width: 100%;
+                > * {
+                    max-width: 100%;
+                    white-space: wrap;
+                }
+                &:last-of-type {
```

Comment:
[issue/suggestion] these kind of selectors are very brittle, I strongly suggest to avoid using them in this way (much simpler and clearer to use a class)